lib/modules/showImages.js

```diff
@@ -323,7 +323,17 @@ addModule('showImages', function(module, moduleID) {
 	var viewImageButton;
 
 	function createImageButtons() {
-		var mainMenuUL = RESUtils.getHeaderMenuList();
+		var mainMenuUL = $('#header-bottom-left ul.tabmenu')[0];
+		// Create new tabmenu on these pages, regardless if one already exists.
+		if ((RESUtils.regexes.search.test(location.href)) ||
+			(/\/about\/(?:reports|spam|unmoderated)/.test(location.href)) ||
+			(location.href.indexOf('/modqueue') !== -1) ||
+			(location.href.toLowerCase().indexOf('/dashboard') !== -1))
+		{
+			mainMenuUL = RESUtils.createElement('ul', null, 'tabmenu viewimages');
+			document.getElementById('header-bottom-left').appendChild(mainMenuUL);
+			mainMenuUL.style.display = 'inline-block'; // Override dashboard's subreddit style.
```

Comment:
It would be nice to fix this in the subreddit stylesheet, and remove the inline style.